db/migrate/20170721141401_create_taggings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTaggings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :taggings do |t|
+      t.integer :tag_id
+      t.integer :taggable_id
+      t.string :taggable_type
+
```

Comment:
@glassjoseph as far as following the conventions go, this looks fine. i'm a little confused though - it looks like "taggings" is just a join table here. why does it need to be polymorphic?